src/Apps/Feature/Components/FeatureMeta.tsx

```diff
@@ -1,19 +1,37 @@
 import React from "react"
-import { Meta, Title } from "react-head"
+import { Link, Meta, Title } from "react-head"
 import { createFragmentContainer, graphql } from "react-relay"
 import { FeatureMeta_feature } from "__generated__/FeatureMeta_feature.graphql"
+import { getENV } from "Utils/getENV"
 
 interface FeatureMetaProps {
   feature: FeatureMeta_feature
 }
 
-const FeatureMeta: React.FC<FeatureMetaProps> = ({ feature: { name } }) => {
-  const TODO_description = "description text"
+const FeatureMeta: React.FC<FeatureMetaProps> = ({
+  feature: { image, metaDescription, name, slug },
+}) => {
+  const description =
+    getENV("CURRENT_PATH") === "/feature/collect-with-artsy"
```

Comment:
I don't think this page exists anymore... (404s, plus I don't see it referenced anywhere in code 🤔), so I think we can remove this conditional.